src/libraries/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -111,6 +111,19 @@ public static void DCS_DateTimeAsRoot()
         Assert.StrictEqual(DataContractSerializerHelper.SerializeAndDeserialize<DateTime>(DateTime.SpecifyKind(DateTime.MaxValue, DateTimeKind.Utc), @"<dateTime xmlns=""http://schemas.microsoft.com/2003/10/Serialization/"">9999-12-31T23:59:59.9999999Z</dateTime>"), DateTime.SpecifyKind(DateTime.MaxValue, DateTimeKind.Utc));
     }
 
+
+    [Fact]
+    public static void DCS_BinarySerializationOfDateTime()
+    {
+        DateTime dateTime = DateTime.Parse("2021-01-01");
+        MemoryStream ms = new();
+        DataContractSerializer dcs = new(dateTime.GetType());
+        using (XmlDictionaryWriter writer = XmlDictionaryWriter.CreateBinaryWriter(ms, null, null, ownsStream: true))
```

Comment:
Do you want to specify `ownsStream: true`? This causes leaving the using to dispose the MemoryStream. I just looked at the implementation of MemoryStream and ToArray works fine even when disposed, but we should still be using best practices even in tests otherwise someone might copy the code and then use a different type of stream where it does matter.